github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -261,6 +287,33 @@ presubmits:
             limits:
               memory: "16Gi"
               cpu: "500m"
+  - name: build-vm-image-builder-image
+    always_run: false
+    run_if_changed: "images/bootstrap/.*"
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-kubevirtci-quay-credential: "true"
+    cluster: phx-prow
```

Comment:
Do we want to use this cluster for new jobs? I thought this would be turned off at some point?